config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gcp-gpu-presubmits.yaml

```diff
@@ -39,6 +39,11 @@ presubmits:
         - --build=quick
         - --cluster=
         - --extract=local
+        # Note: The GCE Node image used may have a dependency on the nvidia-driver-installer image defined in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml
+        # If updating the image defined here, the cos-gpu-installer image may need to updated to support the corresponding COS image.
+        - --env=KUBE_NODE_OS_DISTRIBUTION=gci
+        - --env=KUBE_GCE_NODE_IMAGE=cos-97-16919-189-5
```

Comment:
can we bump to the later family? COS-97 EOL is very close

cc @mmiranda96

Reply:
ack, we should do it. Let's handle in a followup since it also requires updating the installer.